storm-core/src/jvm/backtype/storm/scheduler/Cluster.java

```diff
@@ -480,6 +480,27 @@ private String getStringFromStringList(Object o) {
     * */
     private Double getAssignedMemoryForSlot(Map topConf) {
         Double totalWorkerMemory = 0.0;
+        final Integer TOPOLOGY_WORKER_DEFAULT_MEMORY_ALLOCATION = 768;
+
+        String topologyWorkerGcChildopts = null;
```

Comment:
Do not see any harm of this null assignment, more of a habitual choice. Same below.